test/e2e/api_bank_accounts_test.go

```diff
@@ -13,236 +13,237 @@ import (
 	"github.com/google/uuid"
 
 	"github.com/formancehq/payments/pkg/testserver"
-	. "github.com/formancehq/payments/pkg/testserver"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 )
 
-var _ = Context("Payments API Bank Accounts", func() {
-	var (
-		db  = UseTemplatedDatabase()
-		ctx = logging.TestingContext()
-
-		accountNumber   = "123456789"
-		iban            = "DE89370400440532013000"
-		createRequest   v3.BankAccountsCreateRequest
-		v2createRequest v2.BankAccountsCreateRequest
-
-		app *utils.Deferred[*testserver.Server]
-	)
-
-	app = testserver.NewTestServer(func() Configuration {
-		return Configuration{
-			Stack:                 stack,
-			PostgresConfiguration: db.GetValue().ConnectionOptions(),
-			TemporalNamespace:     temporalServer.GetValue().DefaultNamespace(),
-			TemporalAddress:       temporalServer.GetValue().Address(),
-			Output:                GinkgoWriter,
-		}
-	})
-
-	createRequest = v3.BankAccountsCreateRequest{
-		Name:          "foo",
-		AccountNumber: &accountNumber,
-		IBAN:          &iban,
-	}
-	v2createRequest = v2.BankAccountsCreateRequest{
-		Name:          "foo",
-		AccountNumber: &accountNumber,
-		IBAN:          &iban,
-	}
-
-	When("creating a new bank account with v3", func() {
+var _ = Describe("Payments API Bank Accounts", func() {
```

Comment:
the cleanup function was being called at the wrong time, so I added a wrapper, but the content of the test doesn't really change